src/zope/app/apidoc/static.py

```diff
@@ -465,6 +483,18 @@ def _create_arg_parser():
         """
     )
 
+    ret_kind.add_argument(
+        '--custom-publisher', '-c', dest='ret_kind',
+        help="""Use the publisher directly to retrieve the data and specify
+        a custom ZCML file to load in the form of the.package[:the_file.zcml].
+        The package name and ZCML filename relative to the package are separated by a colon.
+        If the ZCML filename is omitted, `configure.zcml` is assumed.
+        The specified ZCML file needs to include
+        `<include package='zope.app.apidoc' file='ftesting.zcml' condition='have static-apidoc' />`
```

Comment:
(If you accept my suggestion of naming this `static.zml`, this will need to be updated too.)